Dockerfile

```diff
@@ -378,7 +378,7 @@ exec dlv exec /usr/bin/buildkitd \\
   --continue \\
   -- "\$@"
 EOF
-ENV DELVE_PORT 5000
+ENV DELVE_PORT=5000
```

Comment:
nit: these changes could be in another PR

Reply:
💯 -- sorry about that, those were driving me nuts